### DISCUSSION TITLE
AI Search

### DISCUSSION THREAD

**Discussion by Shubham-EV** [fuma-nama/fumadocs#889] [Ideas]
![image](https://github.com/user-attachments/assets/faa8a0cc-83de-4306-8df2-376162090e7a)

This is a dope feature, but it would be nice if it were integrated with the default search UI :)

**Comment by fuma-nama** [comment]
It requires cloud services (not totally free) and isn't easy to setup, built-in integration is in progress but please don't expect it to be useful
